README.md

```diff
@@ -1,24 +1,275 @@
 
-* System dependencies
+After you're finished please remove all the comments and instructions!
+-->
 
-* Configuration
+<div align="center">
 
-* Database creation
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
 
-* Database initialization
+ <img src="https://badgen.net/builder#badge/Ruby%20on%20Rails/7.0.4/red" alt="logo" width="120"  height="auto" />
+<img src="https://img.shields.io/badge/postgres-%23316192.svg?style=for-the-badge&logo=postgresql&logoColor=white" alt="logo" width="120"  height="auto" />
+<img src="https://img.shields.io/badge/github-%23121011.svg?style=for-the-badge&logo=github&logoColor=white" alt="logo" width="120"  height="auto" />
+<img src="https://img.shields.io/badge/ruby-%23CC342D.svg?style=for-the-badge&logo=ruby&logoColor=white" alt="logo" width="120"  height="auto" />
+</div>
 
-* How to run the test suite
+<!-- TABLE OF CONTENTS -->
 
-* Services (job queues, cache servers, search engines, etc.)
+# 📗 Table of Contents
 
-* Deployment instructions
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Run tests](#run-tests)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
 
-* ...
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Budget App 🌍<a name="about-project"></a>
+
+**Budget App** is about building a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+## 🛠 Built With <a name="built-with"></a>
+- Major languages: Ruby
+- Frameworks: Ruby on Rails, Capybara
+- Library: CanCanCan
+- Technologies used: Rubocop, Gem, Postgresql
+- Code Editor: Visual Studio
+- Testing: Rspec
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">RubyonRails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+- **Authorization of users**
+- **Adding of categories and expenses**
+- **Authentication of users**
+
+
+<p align="center">
+  <img src="erd_diagram.png" alt="ERD diagram" />
+</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
```

Comment:
- [ ] Ad it is a requirement for this project, please make sure your deployed project is working properly, checkout the screenshot below and kindly fix the issue. 

![heroku_issue](https://user-images.githubusercontent.com/62849872/204083162-3f816a85-8bdc-4227-bae5-48e57bc1e97c.PNG)